tests/components/websocket_api/test_commands.py

```diff
@@ -6,14 +6,14 @@
 import pytest
 import voluptuous as vol
 
-from homeassistant.bootstrap import SIGNAL_BOOTSTRAP_INTEGRATONS
 from homeassistant.components.websocket_api import const
 from homeassistant.components.websocket_api.auth import (
     TYPE_AUTH,
     TYPE_AUTH_OK,
     TYPE_AUTH_REQUIRED,
 )
 from homeassistant.components.websocket_api.const import URL
+from homeassistant.const import SIGNAL_BOOTSTRAP_INTEGRATONS
```

Comment:
I'm not sure why this change slipped into this PR

Reply:
The constant was moved in part1 (63898) to fix a circular import, this just updates the import.